### PR TITLE
Add survey service deployment pipeline

### PR DESCRIPTION
**Pull request by gavinleeedwards**
# Motivation and Context
Add deployment pipeline for to deploy survey service via Concourse

# What has changed
New survey service pipeline

# How to test?
Fly the pipeline with the pipeline secrets.  Create a new release.  If the pre-production box is ticked then the release will be deployed to prepared.  To deploy to prod, untick the box and run the pipeline again.

### CODE REVIEW

**Comment by necrophonic** [comment]
Added new PR which updates this branch to work with updated `rm-survey-service` code: https://github.com/ONSdigital/ras-deploy/pull/110.

**Review by necrophonic — CHANGES_REQUESTED** [review]
Looks good - should merge in PR https://github.com/ONSdigital/ras-deploy/pull/110 for dealing with new `rm-survey-service` code

**Review by ChristopherMShelton — APPROVED** [review]

File: pipelines/deployment.yml [hunk 1]
```diff
@@ -4025,22 +4016,19 @@ disabled-jobs:
         inputs:
           - name: rm-survey-service-source
         outputs:
-          - name: rm-survey-service-build
-            path: rm-survey-service
+          - name: survey-service
```
Comment (necrophonic) [inline pipelines/deployment.yml]: Do the preprod and prod jobs need to exist here are the "real" ones are separate jobs now (in `pipelines/survey-service.yml `)?

File: pipelines/deployment.yml [hunk 2]
```diff
@@ -4025,22 +4016,19 @@ disabled-jobs:
...
-            make -C /go/src/github.com/ONSdigital/rm-survey-service
-            cp -r /go/src/github.com/ONSdigital/rm-survey-service/ .
+            cp -r rm-survey-service-source/* survey-service
+            make -C survey-service
   - put: cf-resource-prod
     params:
       current_app_name: rm-survey-service-concourse-prod
-      manifest: rm-survey-service-source/manifest.yml
-      path: rm-survey-service-build
+      manifest: survey-service/manifest.yml
```
Comment (necrophonic) [inline pipelines/deployment.yml]: Not sure if we need these jobs (as in previous comment) but these manifests would be wrong I think as there are specific manifests for preprod and prod

**Comment by gavinleeedwards** [comment]
I've now removed all of the preprod and prod jobs that were disabled and reflown the pipeline, so the `deployment.yml` is now around half of the size it was.  It looks as though disabled jobs are ignored anyway so it's made no difference to the pipeline, which is good.

**Review by necrophonic — APPROVED** [review]

**Pull request merged**